starcheck/src/starcheck

```diff
@@ -3,12 +3,12 @@
 unset LD_LIBRARY_PATH
 unset SYBASE
 unset PYTHONPATH
-SKA_bin=`dirname $0`
-eval `${SKA_bin}/flt_envs -shell sh`
+eval `${SKA}/bin/flt_envs -shell sh`
```

Comment:
Oops.  For the use case of SOT MP calling /proj/sot/ska/bin/starcheck, this dirname/SKA_bin thing has the effect of setting SKA reasonably for the rest of the process, without assuming that SKA is already set.